src/lib/components/LoginButton.svelte

```diff
@@ -29,7 +29,7 @@
   on:click={() => login(redirectTo, provider.type)}
 >
   {#if imageSrc}
-    <img src={imageSrc} alt={provider.imageAlt} class="h-8 mr-2" />
+    <img src={imageSrc} alt={provider.imageAlt} class="mr-2" style="height: 2em" />
```

Comment:
```suggestion
    <img src={imageSrc} alt={provider.imageAlt} class="mr-2 h-8" />
```

Reply:
I did this and it seemed to have some issues. I will try it again. It seemed like the h-8 didn't get applied. I will re-test.